test/integration/helpers/Npm.cs

```diff
@@ -0,0 +1,73 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    internal class Npm
+    {
+
+        public static string GetNpmPrefixPath()
+        {
+
+            string npmPath = "npm";
+
+            if (Platform.CurrentPlatform.IsPlatformType(PlatformType.Windows))
```

Comment:
why we only do it for windows?

Reply:
I get it. What I wanted to say is that `npm` might also not exist on other systems if it is not installed or not added to PATH

Reply:
Which might result into a weird generic exception from process builder afterwards